src/ServicePulse.Host/vue/src/assets/main.css

```diff
@@ -275,7 +275,7 @@ pre {
 }
 
 .loading {
-    background-image: url('../img/loader_spinner.gif');
+    background-image: url('../a/img/loader_spinner.gif');
```

Comment:
Since this is the main.css for Vue it feels awkward to reference an image from the angular website.  We are eventually going to be removing all of the angular stuff, so I think it would be better for us to copy the image that we need to the Vue assets folder.  This exact line has already been updated in the feature branch to '../assets/loader_spinner.gif'

Reply:
There are a many such image files in the vue main css. I was planning to have all of them copied over late on. For now, I have copied the images of the ones mentioned in this PR .